legion/sdk/setup.py

```diff
@@ -13,32 +13,46 @@
 #    See the License for the specific language governing permissions and
 #    limitations under the License.
 #
-import json
+import configparser
 import os
 import re
 import typing
 
-from setuptools import setup, find_namespace_packages
+from setuptools import find_namespace_packages, setup
 
 PIPFILE_DEP_SECTION = 'default'
 PACKAGE_ROOT_PATH = os.path.dirname(os.path.abspath(__file__))
-PIP_FILE_LOCK_PATH = os.path.join(PACKAGE_ROOT_PATH, 'Pipfile.lock')
+PIP_FILE_PATH = os.path.join(PACKAGE_ROOT_PATH, 'Pipfile')
 VERSION_FILE = os.path.join(PACKAGE_ROOT_PATH, 'legion/sdk', 'version.py')
 
 
+def convert_dep_version(name: str, version: str) -> str:
+    """
+    Transform python dependency version from pipenv to pip format
+
+    :param name: package name
+    :param version: pipenv version of dependency
+    :return: pip version of dependency
+    """
+    version = version.replace('"', '')
+    version = "" if version == "*" else version
+
+    return f'{name}{version}'
+
+
 def extract_requirements() -> typing.List[str]:
     """
     Extracts requirements from a pip formatted requirements file.
 
     :return: package names as strings
     """
-    with open(PIP_FILE_LOCK_PATH, 'r') as pip_file_lock_stream:
-        pip_file_lock_data = json.load(pip_file_lock_stream)
-        pip_file_section_data = pip_file_lock_data.get(PIPFILE_DEP_SECTION, {})
+    with open(PIP_FILE_PATH, 'r') as f:
```

Comment:
Why did you left Pipfile conversion only for this package?

Reply:
Thanks for cathcing, fixed.